v2/tools/generator/internal/codegen/embeddedresources/renamer.go

```diff
@@ -224,6 +247,7 @@ func makeContextualTypeName(original astmodel.TypeName, context string, suffix s
 	return astmodel.MakeTypeName(original.PackageReference, original.Name()+context+suffix+count)
 }
 
+/*
 func parseContextualTypeName(name astmodel.TypeName) (embeddedResourceTypeName, error) {
```

Comment:
Probably can just delete this? Unless there's a compelling reason to keep it.

Reply:
Agreed.